packages/gasket-plugin-fastify/index.js

```diff
@@ -30,13 +30,17 @@ module.exports = {
     // eslint-disable-next-line max-statements
     createServers: async function createServers(gasket, serverOpts) {
       const fastify = require('fastify');
+      const middie = require('middie');
       const cookieParser = require('cookie-parser');
       const compression = require('compression');
 
       const { config } = gasket;
       const excludedRoutesRegex = config.fastify && config.fastify.excludedRoutesRegex;
       const app = fastify();
 
+      // Enable middleware for fastify@3
+      await app.register(middie);
```

Comment:
Having middie add as a default by the plugin seems fine. Should we also provide a lifecycle (e.g. `fastifyRegister`) for users to register additional modules?

Reply:
Users can already do this via the `fastify` lifecycle down on line 67